freqtrade/commands/cli_options.py

```diff
@@ -658,6 +658,16 @@ def __init__(self, *args, **kwargs):
         nargs='+',
         default=[],
     ),
+    "analysis_date_start": Arg(
+        "--analysis-date-start",
+        help=("Start date to filter trades for analysis (inclusive). "
+              "e.g. '20220101'"),
+    ),
+    "analysis_date_end": Arg(
+        "--analysis-date-end",
+        help=("End date to filter trades for analysis (exclusive). "
+              "e.g. '20220131'"),
+    ),
```

Comment:
is there any reason we can't stick with what we got (`--timerange`) - but need new arguments that are exclusive to this (but seemingly do the same exact thing, with almost the identical syntax).

Reply:
Erm, good question 😄

Reply:
OK, done. Thanks - not sure why I didn't think of that!